cost/downsize_instance/downsize-instance-add-tags.pt

```diff
@@ -0,0 +1,114 @@
+name "Downsize Instances Add Tags Policy Template"
+rs_pt_ver 20180301
+type "policy"
+short_description "A policy that checks cooldown time and adds back in time"
+long_description "Version: 1.0"
+severity "medium"
+category "Cost"
+
+permission "perm_instance_and_monitoring_metrics" do
+  label "Permissions for instances and monitoring metrics"
+  actions "rs_cm.index", "rs_cm.show", "rs_cm.data"
+  resources "rs_cm.instances", "rs_cm.monitoring_metrics"
+end
+
+parameter "param_instance_tag_list" do
+  type "list"
+  label "Instance tags used to filter instances that must validate policy. Example: rs_monitoring:resize=1"
+  min_length 1
+  allowed_pattern /([\w]?)+\:([\w]?)+\=([\w]?)+/
+end
+
+parameter "param_escalate_to" do
+  type "list"
+  label "Email address to send escalation emails to"
+end
+
+auth "auth_rs", type: "rightscale"
+
+resources "clouds", type: "rs_cm.clouds"
+
+resources "instances", type: "rs_cm.instances" do
+  iterate @clouds  
+  cloud_href href(iter_item)
+  filter do
+    state "operational"
+  end
+  tags all("rs_downsize:cooldown=*", "rs_agent:type=right_link_lite")
+end
+
+datasource "ds_instances" do
```

Comment:
Just an fyi... You can name a datasource the same as a resource. If that is confusing to read you can follow this approach. It seems fine and I'm not asking you to change it.

Reply:
I like to prepend it, in my mind it makes it clearer to read.